app/models/task.rb

```diff
@@ -12,7 +12,9 @@ class Task < ActiveRecord::Base
   scope :on_user, -> { where(user: User.current_user) }
 
   def done
-    update(is_done: true) unless is_done
+    unless is_done
```

Comment:
__[rubocop]__
* Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)
* Style/IfUnlessModifier: Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)